File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/Home_ticketStatus.scala
package io.swagger.server.model


/**
 * Ticket statuses.
 *
 * @param Open Count of open tickets. for example: ''4''
 * @param On Hold Count of tickets on hold. for example: ''5''
 */
case class Home_ticketStatus (
  Open: Option[Int],
  On Hold: Option[Int]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/HomeTicketStatus.js
goog.provide('API.Client.Home_ticketStatus');

/**
 * Ticket statuses.
 * @record
 */
API.Client.HomeTicketStatus = function() {}

/**
 * Count of open tickets.
 * @type {!number}
 * @export
 */
API.Client.HomeTicketStatus.prototype.open;

/**
 * Count of tickets on hold.
 * @type {!number}
 * @export
 */
API.Client.HomeTicketStatus.prototype.onHold;

